src/app/pages/send/send-crypto-asset-preview/components/send-crypto-asset-details.layout.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box, Stack, StackProps, Text } from '@stacks/ui';
+
+interface SendCryptoAssetDetailsLayoutProps extends StackProps {
+  amount: string;
+  symbol: string;
+}
```

Comment:
use `Money`?

Reply:
Ok, yeah, I can create a new Money and pass it that way.